publish_push_helper.go

```diff
@@ -0,0 +1,268 @@
+package pubnub
+
+import (
```

Comment:
Should this `import` be removed if we're not importing anything?

Reply:
removed